webpack/global_index.js

```diff
@@ -30,3 +31,4 @@ addGlobalFill(
   'Content View Details',
   <ContentViewDetailsCard key="content-view-details" />,
 );
+addGlobalFill('details-cards', 'Installable errata', <ErrataOverviewCard key="errata-overview" />);
```

Comment:
```suggestion
  <ContentViewDetailsCard key="content-view-details" />,
  2000,
);
addGlobalFill('details-cards', 'Installable errata', <ErrataOverviewCard key="errata-overview" />, 1900);
```
Adding these weights will put our cards ahead of Recent Jobs. thoughts?

Reply:
update: see also https://github.com/theforeman/foreman/pull/8995